src/vcf/store_parse_policy.cpp

```diff
@@ -166,6 +166,8 @@ namespace ebi
             throw new PositionBodyError{state.n_lines};
         }
 
+        check_sorted(state, position);
```

Comment:
I'm not sure I understand. What do you mean with semantic chrom errors? where is that? As I see it, there's only syntax errors (ragel regex) and semantic warnings (meta description) for chrom. the "unsorted error" is treated like a semantic error, and it's done where all the semantic errors are checked, store_parse_policy.

The duplicates check is (already) in the ragel code because of the corner case of returning several errors from the same line. (It is in the ragel part because it can not go into their father ParserImpl because it doesn't know anything about policies, and the children ParserImpl_v4* do. BTW, now I see that the code could be extracted into a method in the ParserImpl_v4* headers 3 times, instead of copied in the ragels 3 times, but it's not a big improvement).